docs/source/quickstart-docker.rst

```diff
@@ -1,49 +1,60 @@
------------------------------------
+.. important::
+    We strongly recommend that you follow the `Zero to JupyterHub`_ tutorial to
+    install JupyterHub.
+   
+   
+Prerequisites
+-------------------------------
+You should have `Docker Desktop`_ installed on a Linux/Unix based system.
 
-A ready to go `docker image <https://hub.docker.com/r/jupyterhub/jupyterhub/>`_
-gives a straightforward deployment of JupyterHub.
 
-.. note::
+Run the Docker Image
+-------------------------------
 
-    This ``jupyterhub/jupyterhub`` docker image is only an image for running
-    the Hub service itself. It does not provide the other Jupyter components,
-    such as Notebook installation, which are needed by the single-user servers.
-    To run the single-user servers, which may be on the same system as the Hub or
-    not, Jupyter Notebook version 4 or greater must be installed.
+To pull the latest JupyterHub image and start the `jupyterhub` container, run this command in your terminal::
+   
+    docker run -d -p 8000:8000 --name jupyterhub jupyterhub/jupyterhub jupyterhub
 
-Starting JupyterHub with docker
--------------------------------
+This command exposes the Jupyter container on port:8000. Navigate to `http://localhost:8000` in a web browser to access the JupyterHub console.
 
-The JupyterHub docker image can be started with the following command::
+You can stop and resume the container by running `docker stop` and `docker start` respectively.::
 
-    docker run -d -p 8000:8000 --name jupyterhub jupyterhub/jupyterhub jupyterhub
+    # find the container id
+    docker ps
 
-This command will create a container named ``jupyterhub`` that you can
-**stop and resume** with ``docker stop/start``.
+    # stop the running container
+    docker stop  <container-id>
 
-The Hub service will be listening on all interfaces at port 8000, which makes
-this a good choice for **testing JupyterHub on your desktop or laptop**.
+    # resume the paused container
+    docker start <container-id>
 
-If you want to run docker on a computer that has a public IP then you should
-(as in MUST) **secure it with ssl** by adding ssl options to your docker
+
+If you are running Docker on a computer that has a public IP address, you must **secure it with ssl** by adding ssl options to your docker
 configuration or using a ssl enabled proxy.
 
-`Mounting volumes <https://docs.docker.com/engine/admin/volumes/volumes/>`_
-will allow you to store data outside the docker image (host system) so it will
-be persistent, even when you start a new image.
 
-The command ``docker exec -it jupyterhub bash`` will spawn a root shell in your
-docker container. You can use the root shell to **create system users in the container**.
-These accounts will be used for authentication in JupyterHub's default
+`Mounting volumes <https://docs.docker.com/engine/admin/volumes/volumes/>`_ 
+persist and stores the data generated by the docker container, even when you stop the container. 
```

Comment:
Verb-noun agreement: the subject is the gerund `mounting`, which "persists and stores" the data, or what we had before, _enables you_ to persist and store...